docs/source/release_notes/52_0.rst

```diff
@@ -0,0 +1,139 @@
+========
+52.0 LTS
+========
+
+The Avocado team is proud to present another release: Avocado version
+52.0, the second Avocado LTS version.
+
+Release documentation: `Avocado 52.0
+<http://avocado-framework.readthedocs.io/en/52.0/>`_
+
+When compared to the last LTS (v36), the main features introduced by
+this versions are:
+
+- TAP[2] version 12 support.
+- The download of assets in tests now allow for an expiration time.
+- Environment variables can be propagated into tests running on remote
+  systems.
+- The plugin interfaces have been moved into the
+  `avocado.core.plugin_interfaces` module.
+- A contrib script for running `kvm-unit-tests`.
+- Support for running tests in Docker container.
+- Introduction of the “Fail Fast” feature.
+- Special keyword `latest` for replaying previous jobs.
```

Comment:
plus also the support to replay results by path (verified it was not possible in 36)